src/main/java/com/hollingsworth/arsnouveau/common/entity/BubbleEntity.java

```diff
@@ -61,22 +63,23 @@ public void registerControllers(AnimatableManager.ControllerRegistrar controller
     @Override
     public void tick() {
         super.tick();
-        if(!level.isClientSide){
+
+        if (entityData.get(HAS_POPPED)) {
+            poppingTicks++;
+        } else if(!level.isClientSide) {
```

Comment:
Changing this to an elseif is a bug, poppingTicks is needed on both sides and the below logic should still run.